styles.css

```diff
@@ -1,215 +1,294 @@
   font-size: 18px;
   font-style: normal;
   font-variant: normal;
   font-weight: 400;
   line-height: 24px;
-  margin-top: 10px;
   margin-bottom: 10px;
+  max-width: 260px;
 }
 
-.content{
-  margin-top: 60px;
-  margin-left: 60px;
+.site__header__logo img {
+  height: auto;
+  width: 160px;
 }
 
-.entry{
+/* BODY SECTION
+	-------------------------------- */
+.entry {
+  display: flex;
   margin-top: 50px;
-  margin-bottom: 50px;
+  margin-bottom: 70px;
 }
 
-.top_entry{
-	display: flex;
-	flex: 0 0 200px;
+.entry__number {
+  flex-basis: 45px;
 }
 
-.number{
-
+.entry__text h2 {
+  align-items: center;
+  display: flex;
+  justify-content: flex-start;
+  white-space: wrap;
 }
 
-.title{
+.entry__text .author,
+.entry__text .author__mobile {
+  background-color: #fff;
+  border: 1px solid #000;
+  border-radius: 6px;
+  display: inline-block;
+  font-size: 14px;
+  line-height: 1;
+  margin-left: 15px;
+  padding: 4px 8px;
 }
 
-.description{
+.entry__text .author__mobile {
+  display: none;
+  margin-left: 0;
 }
 
-.author{
-	text-align: center;
-	width: auto;
-	height: 25px;
-	background: #FFFFFF;
-	border: 1px solid #000000;
-	box-sizing: border-box;
-	border-radius: 5px;
-	padding: 10px;
+.entry__text p {
+  max-width: 260px;
 }
 
-/* COMMON STYLES
-  -------------------------------- */
+/* FOOTER
+	-------------------------------- */
+.site__footer {
+  padding-bottom: 80px;
+}
 
-	/* LAYOUT */
-		/* {position:absolute;} */
-		/* {position:relative;} */
-		/* {float:left;} */
-		/* {float:right;} */
-		/* {overflow:hidden;} */
-		/* {display:block;} */
-		/* {display:inline;} */
-		/* {display:none;} */
-		/* {margin:0} */
-		/* {padding:0;} */
-		/* {width:100%;} */
-
-	/* TYPOGRAPHY */
-		/* {line-height:1;} */
-		/* {list-style-type:none;} */
-		/* {font-size:1em;} */
-		/* {font-style:italic;} */
-		/* {font-style:normal;} */
-		/* {font-weight:bold;} */
-		/* {font-weight:normal;} */
-		/* {text-align:center;} */
-		/* {text-align:right;} */
-		/* {text-align:left;} */
-		/* {text-decoration:underline;} */
-		/* {text-decoration:none;} */
-		/* {text-transform:uppercase;} */
-		/* {text-transform:none;} */
-
-	/* COLOR */
-		/* {background-color: #fff;} */
-		/* {background-color: transparent;} */
-		/* {border: 1px solid #000;} */
-		/* {border: none;} */
-		/* {color: #000;} */
-		/* {border-radius: 0px;} */
-		/* {transition: all 0.3s ease-in-out;} */
+.site__footer p {
+  margin-bottom: 40px;
+}
 
+/* MOBILE STYLES
+  -------------------------------- */
+/* Smaller than Desktop: */
+@media screen and (max-width: 1025px) {
+}
+/* Smaller than Laptop: */
+@media screen and (max-width: 770px) {
+  .entry__text h2 {
+    display: block;
+  }
+  .entry__text .author {
+    display: none;
+  }
+  .entry__text .author__mobile {
+    display: inline-block;
+    margin-top: 15px;
+  }
+  .site__header__logo img {
+    width: 120px;
+  }
+}
+/* Smaller than Tablet: */
+@media screen and (max-width: 480px) {
+}
+/* LAYOUT */
```

Comment:
All of this stuff below should be deleted - super unneeded but I kept it just bc it is not my codebase